CMakeLists.txt

```diff
@@ -267,10 +261,10 @@ if(AMENT_ENABLE_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-  file(
-    COPY "${PROJECT_NAME}"
-    DESTINATION "${CMAKE_CURRENT_BINARY_DIR}"
-  )
+file(
+  COPY "${PROJECT_NAME}"
```

Comment:
I don't know, I was just fixing the indention. It looks like it is copying Python stuff from the source directory to the build directory.